config/data/langs.toml

```diff
@@ -883,6 +883,29 @@ example = '''
 `0:x
 '''
 
+[Lil]
+args       = [ '/usr/bin/lilt', '-e', '$code' ]
+experiment = 1174
+size       = '1.67 MiB'
+version    = 'Lilt 1.48'
+website    = 'https://beyondloom.com/decker/lilt.html'
+example    = '''
+# Printing
+print["Hello, World!"]
+
+# Looping
+each i in range 10
+    print[i]
+end
+
+# Accessing arguments
+each i in range count args
+  if i > 2
+    print[args[i]]
+  end
+end
```

Comment:
Using the binary primitive `drop`:
```
each arg in 2 drop args
    print[arg]
end
```

Reply:
Nice, that's way better. It was my intention to do something like that, but lost patience while I couldn't figure it out.
However, it's still not the best of the best. Let me show you.

The reason why I wrote `i > 2` is because the program seems to interpret exactly each entry on the command-line as an argument, starting from the entry point. If we only drop two arguments, the supplied source code is Lil's third argument.

```bash
oh@Yewzir:~/code-golf$ docker run -it --rm codegolf/lang-lil -e "each arg in 2 drop args print[arg] end" abc def ghi 123 456 789
each arg in 2 drop args print[arg] end
abc
def
ghi
123
456
789
```

For that reason, three arguments need to be dropped from the list.

```bash
oh@Yewzir:~/code-golf$ docker run -it --rm codegolf/lang-lil -e "each arg in 3 drop args print[arg] end" abc def ghi 123 456 789
abc
def
ghi
123
456
789
```

Rebol behaves practically identical, with the exception that it can be prevented with specific command-line options.

I will push an update as soon as possible, plus I just thought of something else too.

Thanks, pal! Good lookin' out.